yaptide/celery/tasks.py

```diff
@@ -56,8 +56,9 @@ def run_single_simulation(self,
                           task_id: str,
                           update_key: str = None,
                           simulation_id: int = None,
-                          keep_tmp_files: bool = False) -> dict:
-    """Function running single shieldhit simulation"""
+                          keep_tmp_files: bool = False,
+                          sim_type: str = 'shieldhit') -> dict:
+    """Function running single simulation"""
```

Comment:
I am not sure if we want to add parameters here.
Maye, each new simulator should have its own task?

Reply:
If the code would be very different then it may be a good idea